src/components/Register/AccountDetails/SaveAccount/SaveAccount.js

```diff
@@ -52,9 +64,32 @@ export default class SaveAccount extends React.Component {
     this.props.setLabel(event.target.value);
   }
 
-  handleSave = () => {
+
+  handleAddToWallet = () => {
+    const { account, label } = this.props;
+
+    const filenames = remote.dialog.showOpenDialog({
+      title: 'Add account to a NEP6 Wallet',
+      message: 'Add account to a NEP6 Wallet',
+      filters: [{ name: 'NEP6 Wallet File', extensions: ['json'] }]
+    });
+
+    if (filenames) {
```

Comment:
:neckbeard: To prevent nested `if`'s, it might be a little cleaner to return earlier here.  i.e.:

```js
if (!filenames) {
  return;
}
```